common/app/views/fragments/atoms/youtube.scala.html

```diff
@@ -58,7 +58,7 @@
 
         @for(asset <- activeAsset if playable && !expired) {
 
-            <div id="youtube-@asset.id" data-asset-id="@asset.id" class="youtube-media-atom__iframe"></div>
+            <iframe id="youtube-@asset.id" data-asset-id="@asset.id" class="youtube-media-atom__iframe" src="https://www.youtube-nocookie.com/embed/@{asset.id}?enablejsapi=1"></iframe>
```

Comment:
@regiskuckaertz I have a question regarding swapping out the placeholder for an `iframe`. Currently when we initialise the `new window.YT.Player` (here:  https://github.com/guardian/frontend/blob/fd4256415a69a9911f62ae6fc2761979763df9fd/static/src/javascripts/projects/common/modules/atoms/youtube-player.js#L108) we pass `eltId ` which is the placeholder `div`'s ID. I believe the Youtube script then inserts an iframe into this placeholder, or replaces it with an iframe. 

For performance reasons on fronts we now wait to set up the player until the user clicks the placeholder, this means we wait to call `new window.YT.Player` which in turn inserts the `iframe`. The reason this was done was so we didn't preload multiple `iframe`s in the video carousel. the following PR made that change: https://github.com/guardian/frontend/pull/20286

Now we have the iframe there by default will we be loading all these `iframe`s and there assets to begin?

Reply:
Yes 😒

Reply:
@regiskuckaertz You can simply configure the `host` using `window['YTConfig'].host = "www.youtube-nocookie.com"` creating `YTConfig` if it does not exist.

Reply:
This way you can keep the `placeholder` and only update `youtube-player.js`

Reply:
huhoo, some hidden feature. It actually didn't work but it turns out you can set the `host` setting as a `YT.Player` setting, also undocumented but it worked 🎉 Thanks Mariot!